test/inductor/test_codecache.py

```diff
@@ -80,22 +78,6 @@ def forward(self, x):
 
 @instantiate_parametrized_tests
 class TestFxGraphCache(TestCase):
-    @classmethod
```

Comment:
Not needed any more because I do the same dir mocking in the main test file now, i.e., for all tests